subsys/bluetooth/controller/Kconfig

```diff
@@ -94,6 +94,18 @@ config BT_CTLR_HCI_VS_BUILD_INFO
 	  character is required at the beginning to separate it from the
 	  already included information.
 
+config BT_CTLR_HCI_ADV_HANDLE_MAPPING
+	bool "Enable advertising set handle mapping between HCI and LL"
+	default y if BT_HCI_RAW
+	default n
+	depends on BT_CTLR_ADV_EXT
```

Comment:
nitpick, move this line below the `bool` prompt like, so that `depends on` is close to the `config` even when additions defaults are appended, if any, in the future.